presto-parser/src/main/java/io/prestosql/sql/tree/Identifier.java

```diff
@@ -51,6 +51,7 @@ private Identifier(Optional<NodeLocation> location, String value, boolean delimi
         this.value = value;
         this.delimited = delimited;
 
+        checkArgument(!value.isEmpty(), "value must not be empty");
```

Comment:
`"value is empty"`  would be consistent with `"value is null"` which we use when doing null check.

also use `this.value = requireNonNull(value, "value is null");` above